### DISCUSSION TITLE
Interface Light Mode

### DISCUSSION THREAD

**Discussion by ghost** [OctoFarm/OctoFarm#1339] [Feature Request]
Would it be possible to put a toggle switch in for light mode? Yes, I'm one of those people...

**Comment by NotExpectedYet** [comment]
Howling! aha. 

Yes it's totally possible, I just don't really want to work it into the current implementation as it's not setup for it and it's using an old Bootstrap UI version. It's a lot of work basically, there's custom colours I've used everywhere. Be happy for someone to do a PR for it. 

I have however done a theme builder in the new version, it's going to be in V2.0 though which will be a ways off yet as I'm writing the system from scratch.

**Comment by ghost** [comment]
Ha, dark mode should be the default I'm not a complete menace.
Unfortunately I do not have the skill to contribute though so I will wait patiently for 2.0. Thanks!

**Reply by NotExpectedYet** [reply]
It will be coming sooner. I've just started upgrading too bootstrap 5 (ui library) and it's got a very easy toggle. I'm hoping to get the UI work into 1.8.0 but if not it will be 1.9.0.
